core/src/commonMain/kotlin/com/ustadmobile/core/domain/password/SavePasswordUseCase.kt

```diff
@@ -0,0 +1,5 @@
+package com.ustadmobile.core.domain.password
+
+interface SavePasswordUseCase {
```

Comment:
I think this should have comments to explain that this is saving the password to a password manager (e.g. not being saved internally). The interface name itself is fine.